rosa_architecture/cloud-experts-rosa-hcp-sts-explained.adoc

```diff
@@ -37,13 +37,28 @@ Security features for AWS STS include:
+* *AWS infrastructure* - The infrastructure required for the cluster including the Amazon EC2 instances, Amazon EBS storage, and networking components. See xref:../rosa_architecture/rosa_policy_service_definition/rosa-hcp-service-definition.adoc#rosa-sdpolicy-instance-types_rosa-hcp-service-definition[AWS compute types] to see the supported instance types for compute nodes and xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-ec2-instances_rosa-sts-aws-prereqs[provisioned AWS infrastructure] for more information on cloud resource configuration.
 * *AWS STS* - A method for granting short-term, dynamic tokens to provide users the necessary permissions to temporarily interact with your AWS account resources.
 * *OpenID Connect (OIDC)* - A mechanism for cluster Operators to authenticate with AWS, assume the cluster roles through a trust policy, and obtain temporary credentials from AWS IAM STS to make the required API calls.
 * *Roles and policies* - The roles and policies used by {hcp-title} can be divided into account-wide roles and policies and Operator roles and policies.
 +
-The policies determine the allowed actions for each of the roles. See link:https://docs.openshift/rosa/rosa_architecture/rosa-sts-about-iam-resources.html#rosa-sts-about-iam-resources[About IAM resources for ROSA clusters that use STS] for more details about the individual roles and policies and link:https://docs.openshift/rosa/rosa_planning/rosa-sts-ocm-role.html#rosa-sts-ocm-role[ROSA IAM role resource] for more details about trust policies.
+The policies determine the allowed actions for each of the roles.
+ifdef::openshift-rosa[]
+See xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-about-iam-resources[About IAM resources] for more details about the individual roles and policies. See xref:../rosa_planning/rosa-sts-ocm-role.adoc#rosa-sts-ocm-role[ROSA IAM role resource] for more details about trust policies.
+endif::openshift-rosa[]
+ifdef::openshift-rosa-hcp[]
+See xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-hcp-about-iam-resources[About IAM resources] for more details about the individual roles and policies. See xref:../rosa_planning/rosa-hcp-prepare-iam-roles-resources.adoc#rosa-hcp-prepare-iam-roles-resources[Required IAM roles and resources] for more details on preparing these resources in your cluster.
+endif::openshift-rosa-hcp[]
 +
+--
+** The account-wide roles are:
+
+*** ManagedOpenShift-Installer-Role
```

Comment:
Should this just be Installer, Worker, Support? The ManagedOpenShift piece is just a prefix
Also for HCP clusters the suffix would be `HCP-ROSA-<type>-Role` instead of `<type>-Role`. These are all just expected flows from rosa cli though, unsure if needed to be captured, please validate with PM

Reply:
Possibly, will verify - I believe we have used the generic Installer/Worker/Support elsewhere, but customers tend to prefer more specifics.